recipe-client-addon/bootstrap.js

```diff
@@ -90,9 +92,13 @@ this.shutdown = function(data, reason) {
     "lib/SandboxManager.jsm",
     "lib/Storage.jsm",
   ];
-  for (const module in modules) {
+  for (const module of modules) {
+    log.debug(`Unloading ${module}`);
     Cu.unload(`resource://shield-recipe-client/${module}`);
   }
+
+  // Don't forget the logger!
+  log = null;
```

Comment:
Oh, I misread the diff, and didn't catch that `log` was a module global. Carry on.